lib/bootloader_setup.pm

```diff
@@ -476,16 +510,15 @@ sub specific_bootmenu_params {
             # 'ifcfg=*=dhcp' sets this variable in ifcfg-eth0 as well and we can't
             # have them both as it's not deterministic.
             $netsetup = get_var("NETWORK_INIT_PARAM", "ifcfg=*=dhcp SetHostname=0");
-            # If AUTOYAST is url (://) or slp dont't translate it to openqa asset
-            $autoyast = data_url($autoyast) if $autoyast !~ /^slp$|:\/\//;
-            $args .= " $netsetup autoyast=$autoyast ";
+            $args .= " $netsetup ";
+            $args .= autoyast_boot_params;
         }
         else {
             $netsetup = " " . get_var("NETWORK_INIT_PARAM") if defined get_var("NETWORK_INIT_PARAM");    #e.g netsetup=dhcp,all
             $args .= $netsetup;
         }
     }
-    if (get_var("AUTOUPGRADE")) {
+    if (get_var("AUTOUPGRADE") || get_var("UPGRADE_FROM_AUTOYAST") || get_var("UPGRADE")) {
```

Comment:
does not look like a good idea: https://openqa.opensuse.org/tests/738753#step/welcome/11 fix or revert?